docs/details/advisors/checks/Develop Security Checks

```diff
@@ -0,0 +1,260 @@
+            is_percona = 'psmdbVersion' in info
+ 
+            # parse_version returns a dict with keys: major, minor, patch, rest, num
+            version = parse_version(info["version"])
+            print("version =", repr(version))
+            num = version["num"]
+            mm = "{}.{}".format(version["major"], version["minor"])
+ 
+            results = []
+ 
+            if is_percona:
+                latest = LATEST_VERSIONS["percona"][mm]
+                if latest > num:
+                    results.append({
+                        "summary": "Newer version of Percona Server for MongoDB is available",
+                        "description": "Current version is {}, latest available version is {}.".format(format_version_num(num), format_version_num(latest)),
+                        "severity": "warning",
+                        "labels": {
+                            "current": format_version_num(num),
+                            "latest":  format_version_num(latest),
+                        },
+                    })
+ 
+                return results
+ 
+            if True:  # MongoDB
+                latest = LATEST_VERSIONS["mongodb"][mm]
+                if latest > num:
+                    results.append({
+                        "summary": "Newer version of MongoDB is available",
+                        "description": "Current version is {}, latest available version is {}.".format(format_version_num(num), format_version_num(latest)),
+                        "severity": "warning",
+                        "labels": {
+                            "current": format_version_num(num),
+                            "latest":  format_version_num(latest),
+                        },
+                    })
+ 
+                return results
+```
+
+## Check fields
+
+- **Version** (integer, required) defines what other check properties are expected, what types are supported, what is expected from the script and what it can expect from the execution environment, etc.
+- **Name** (string, required) defined machine-readable name (ID).
+- **Summary** (string, required) defines short human-readable summary.
+- **Description** (string, required) defines long human-readable description.
+- **Type** (string/enum, required) defines PMM Service type for which check is performed and query type.
+- **Query** (string, optional) contains a SQL query or MongoDB query document (as a string with proper quoting) which is executed on the PMM Client side. It may be absent if type defines the whole query by itself.
+- **Script** (string, required) contains a small Python program that processes query results, and returns check results. It is executed on the PMM Server side.
+
+Check script assumes that there is a function with a fixed name *check* that accepts a *list* of *dicts* containing returned rows for SQL databases and documents for MongoDB. It returns zero, one, or several check results that are then converted to alerts.
```

Comment:
Another function that should be implemented is `check_context`. Versions of PMM < 2.12 use `context` and the newer versions use `check_context`.  Both have the same meaning.